tests/conftest.py

```diff
@@ -847,6 +847,12 @@ def get_serialized_data(self):
                 return json.loads(data)
             return data
 
+        def _bag_dag_compat(self, dag):
```

Comment:
Why do we need this? Isn’t the test always run in newer versions? Or is this supposed to be removed when/before we release 3.0 or something?

Reply:
Because of our directory structure, tests in providers inherit this conftest.

Reply:
I see. Maybe a note here for this and a todo to remove this when all providers drop 2.x support?

Reply:
Added